### PR TITLE
fix: clean up setup scripts to support ARM64

### PR DESCRIPTION
**Pull request by kenji-miyake**
## Description

As I wrote in https://github.com/autowarefoundation/autoware/pull/154#issuecomment-1100786584, ARM64 build isn't passing due to the conflicts in CUDA versions.

To resolve this, I changed the code to define env values in `amd64.env` and `arm64.env` and load them in the scripts.

It can also remove duplicates of versions in multiple places.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by kenji-miyake** [comment]
CI -> Failed
- build amd64: https://github.com/autowarefoundation/autoware/runs/6145390013?check_suite_focus=true
- build arm64: https://github.com/autowarefoundation/autoware/runs/6145390950?check_suite_focus=true
- docker-build amd64: https://github.com/autowarefoundation/autoware/runs/6145391210?check_suite_focus=true
- docker-build arm64: https://github.com/autowarefoundation/autoware/runs/6145391446?check_suite_focus=true

**Comment by kenji-miyake** [comment]
TODO -> Fixed

- Remove comments in `arm64.env`.
image

- Allow workflow_dispatch for PRs.
image

- Rename env values.
image

**Comment by kenji-miyake** [comment]
CI
- https://github.com/autowarefoundation/autoware/runs/6145525989?check_suite_focus=true
- https://github.com/autowarefoundation/autoware/runs/6145526463?check_suite_focus=true
- https://github.com/autowarefoundation/autoware/runs/6145526710?check_suite_focus=true
- https://github.com/autowarefoundation/autoware/runs/6145526981?check_suite_focus=true

**Comment by kenji-miyake** [comment]
CI
- https://github.com/autowarefoundation/autoware/runs/6145660803?check_suite_focus=true -> Passed
- https://github.com/autowarefoundation/autoware/runs/6146028939?check_suite_focus=true -> Passed
- https://github.com/autowarefoundation/autoware/runs/6146316864?check_suite_focus=true
- https://github.com/autowarefoundation/autoware/runs/6146064664?check_suite_focus=true -> Passed

File: ansible/roles/rmw_implementation/tasks/main.yaml [hunk 1]
```diff
@@ -1,12 +1,12 @@
-- name: Get kebab-case name
+- name: Get dash-case name of rmw_implementation
```
Comment (kenji-miyake) [inline ansible/roles/rmw_implementation/tasks/main.yaml:1]: Followed this rule.

https://developers.google.com/style/word-list#letter-k
image

**Review by esteve — APPROVED** [review]

**Pull request merged**